This is a quite ugly hack to make it compile.

TODO: use libunwind instead.

---
--- a/tmate-debug.c
+++ b/tmate-debug.c
@@ -60,18 +60,17 @@ void tmate_print_stack_trace(void)
 	char **strings;
 	size_t i;
 
+#if 0
 	size = backtrace (array, 20);
 	strings = backtrace_symbols (array, size);
 
 	tmate_info ("============ %zd stack frames ============", size);
 
 	for (i = 1; i < size; i++) {
-#if DEBUG
 		if (print_resolved_stack_frame(strings[i]) < 0)
-#endif
 			tmate_info("%s", strings[i]);
 	}
-
+#endif
 	free (strings);
 }
 
